cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -192,6 +192,36 @@ def validate_fsx_ignored_parameters(self, data, **kwargs):
             if messages:
                 raise ValidationError(message=messages)
 
+    @validates_schema
+    def validate_backup_id_unsupported_parameters(self, data, **kwargs):
+        """Return errors for parameters in the FSx config section that would be ignored."""
+        # If file_system_id is specified, all parameters are ignored.
+        messages = []
+        if data.get("backup_id") is not None:
+            unsupported_config_param_names = [
+                "deployment_type",
+                "per_unit_storage_throughput",
+                "storage_capacity",
+                "import_path",
+                "export_path",
+                "imported_file_chunk_size",
+                "kms_key_id",
+            ]
+
+            for key in data:
+                if key in unsupported_config_param_names:
+                    messages.append(FSX_MESSAGES["errors"]["unsupported_backup_param"].format(name=key))
+
+            if messages:
+                raise ValidationError(message=messages)
+
+        if data.get("file_system_id") is not None:
```

Comment:
This test is in the `validate_file_system_id_ignored_parameters` why are we duplicating it? I think it's a cut/paste error.